async/mailbox.go

```diff
@@ -105,6 +111,7 @@ func (bx *Mailbox) NewAsyncError(cb AsyncErrorResponseHandler) *AsyncError {
 // the callback function and removes the message from the mailbox
 func (bx *Mailbox) ProcessMessages() {
 	var unCompletedMsgs []message
+	start := time.Now()
```

Comment:
We already capture this in stateful_scheduler.go (stats.SchedProcessMessagesLatency_ms)